server/controllers/routes/admin/login.js

```diff
@@ -0,0 +1,41 @@
+const bcrypt = require('bcrypt');
```

Comment:
You can make destructure here to get compare direct
```suggestion
const  = { compare } require('bcrypt');
```